shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -36,13 +34,16 @@ case class TotalPaxSource(pax: Int, feedSource: FeedSource, splitSource: Option[
     compare(that)
 
   override def compare(that: TotalPaxSource): Int =
-    if (this.pax < that.pax) {
+    if (this.feedSource == that.feedSource && this.splitSource == that.splitSource && this.pax < that.pax) {
```

Comment:
Can you explain why we have implemented `Ordered` on this class?

Reply:
If we look at one of test in DRT FlightUpdatesTriggerNewPortStateSpec where we update arrival from 21 to 50 for same source . I expect totalPax to have 2 entry for LiveSource one with 21 and other with 50 . So i have ordered it so we can pick one with high pax at moment but we can add other field to decide on picking .

Reply:
Let me remove the sortedSet . May be after removing totalPax from `update` def in Arrival we may not have more than one actual pax from same source feed.